tests/Unit/Domain/Creators/Test_Rectangle.cpp

```diff
@@ -123,23 +147,61 @@ void test_rectangle() {
 
 void test_rectangle_factory() {
   INFO("Rectangle factory");
-  const auto domain_creator =
-      TestHelpers::test_factory_creation<DomainCreator<2>>(
-          "Rectangle:\n"
-          "  LowerBound: [0,0]\n"
-          "  UpperBound: [1,2]\n"
-          "  IsPeriodicIn: [True,False]\n"
-          "  InitialGridPoints: [3,4]\n"
-          "  InitialRefinement: [2,3]\n");
-  const auto* rectangle_creator =
-      dynamic_cast<const creators::Rectangle*>(domain_creator.get());
-  test_rectangle_construction(
-      *rectangle_creator, {{0., 0.}}, {{1., 2.}}, {{{3, 4}}}, {{{2, 3}}},
-      std::vector<DirectionMap<2, BlockNeighbor<2>>>{
-          {{Direction<2>::lower_xi(), {0, {}}},
-           {Direction<2>::upper_xi(), {0, {}}}}},
-      std::vector<std::unordered_set<Direction<2>>>{
-          {{Direction<2>::lower_eta()}, {Direction<2>::upper_eta()}}});
+  {
```

Comment:
If the two are separate enough to scope like this, maybe put in some `INFO`s to indicate if a possible test failure is in a time-dependent or time-independent domain. The test failure line numbers might not be informative here because they call the same function just with different arguments. (for the other two domains in this PR also)